python/pyspark/ml/param/_shared_params_code_gen.py

```diff
@@ -115,7 +118,7 @@ def get$Name(self):
         ("outputCol", "output column name", None),
         ("numFeatures", "number of features", None),
         ("checkpointInterval", "checkpoint interval (>= 1)", None),
-        ("seed", "random seed", None),
+        ("seed", "random seed", "random.randint(0, sys.maxsize)"),
```

Comment:
So the original bug mentioned that in Scala it was done randomly. I think for testing using a fixed seed in the constructor is a good way to ensure reproduceability. I'm fine with replacing this with a hash of the class or something if we think that would be better, but I think it probably makes sense to do the same thing between Python & Scala.